src/handlers/notify_zulip.rs

```diff
@@ -45,8 +46,8 @@ pub(super) async fn parse_input(
                 })
                 .map(|input| vec![input]))
         }
-        IssuesAction::Closed | IssuesAction::Reopened => {
-            Ok(Some(parse_close_reopen_input(event, config)))
+        IssuesAction::Opened | IssuesAction::Closed | IssuesAction::Reopened => {
+            Ok(Some(parse_open_close_reopen_input(event, config)))
```

Comment:
I was looking for a slightly more parseable (ha!) name for this function. `parse_input()`? `get_notification_input()`? Suggestions?

Reply:
We already have a `parse_label_change_input` for label and unlabel events, I rather not make less distinctive.

Better have a descriptive function name than a short one that is confusing.